pkg/operator/flags.go

```diff
@@ -10,6 +10,7 @@ const (
 	AzureSubnetsServiceEndpointManaged = "aro.azuresubnets.serviceendpoint.managed"
 	BannerEnabled                      = "aro.banner.enabled"
 	CheckerEnabled                     = "aro.checker.enabled"
+	CPMSDeactivatorEnabled             = "aro.cpmsdeactivator.enabled"
```

Comment:
Does this turn machinesets off or on? Name is confusing can we make it clearer.

Reply:
Happy for better suggestions for the name. 

Enabling the controller disables the machineset, which is inherently confusing.